### ISSUE TITLE
Call log modal doesn't make background content inert while open

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/call.zetk.in#271]
## Steps to reproduce

1. Open an assignment.
2. Click the log button.
3. Press `⌘ + F5` to start VoiceOver.
4. Use `^⌥→` and `^⌥←` to try to interact with the background content behind the modal.

## Expected outcome

It should be impossible to interact with the background content while the modal's open.

![GIF where I fail and can't get to the background content](https://user-images.githubusercontent.com/566159/191598007-38f506f4-3ea3-4893-ab4a-1efcfb8e2784.gif)

## Actual outcome

It's possible to escape the modal and interact with the background content.

![GIF where I succeed and get to read the content that's almost totally obscured by the overlay](https://user-images.githubusercontent.com/566159/191599243-942f3621-2a0f-4c48-9f99-f5001362b9b8.gif)

## Severity

The success criterion covering this issue is WCAG level A – the minimum level. Most organizations aim for a conformance level of AA, which includes all level A criteria. Having background content available while a modal is open can be a a cause for real confusion so the severity of this issue is high.

## Recommendation

In order to create the `Expected outcome` GIF above I just put the `inert` attribute on the `.App-content` div. It could be that that's all it takes to fix this one.
